src/configure

```diff
@@ -796,10 +796,9 @@ echo "CONFIGURE_VERSION := $CONFIGURE_VERSION" >> kaldi.mk
 echo "FSTROOT = $FSTROOT" >> kaldi.mk
 
 # Check installed OpenFst version and add C++11 flags if OpenFst >= 1.4
-OPENFST_VER=`grep 'PACKAGE_VERSION' $FSTROOT/Makefile | sed -e 's:.*= ::'`
+OPENFST_VER="${OPENFST_VER:-`grep 'PACKAGE_VERSION' $FSTROOT/Makefile | sed -e 's:.*= ::'`}"
 echo "OPENFST_VER = $OPENFST_VER" >> kaldi.mk
-OPENFST_VER_NUM=`echo $OPENFST_VER | sed 's/\./ /g' | xargs printf "%d%02d%02d"`
-if [ $OPENFST_VER_NUM -ge 10400 ]; then
+if [ `sed 's/\./0/g' <<< $OPENFST_VER` -ge 10400 ]; then
```

Comment:
Please don't use it, it doesn't seem to be universally supported

mac:~: sed s/a/b/ <<< echo b

sed: b: No such file or directory

mac:~: echo $SHELL



On Tue, Feb 2, 2016 at 6:54 PM, Alexander Solovets <notifications@github.com
> wrote:

> In src/configure
> <https://github.com/kaldi-asr/kaldi/pull/480#discussion_r51657442>:
>
> >  echo "OPENFST_VER = $OPENFST_VER" >> kaldi.mk
> > -OPENFST_VER_NUM=`echo $OPENFST_VER | sed 's/\./ /g' | xargs printf "%d%02d%02d"`
> > -if [ $OPENFST_VER_NUM -ge 10400 ]; then
> > +if [ `sed 's/\./0/g' <<< $OPENFST_VER` -ge 10400 ]; then
>
> Send the following line to stdin.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/480/files#r51657442>.
>